WindowsFormsApp1/FaceRecognitionController.cs

```diff
@@ -35,6 +44,8 @@ public void Initialize()
         }
         int err = 0;
         FSDK.SetTrackerMultipleParameters(tracker, "HandleArbitraryRotations=false; DetermineFaceRotationAngle=false; InternalResizeWidth=300; FaceDetectionThreshold=5;", ref err);
+
+        rectInfoDictionary = new Dictionary<long, Queue<RectInfo>>();
     }
 
     public void DoLoop(FSDK.CImage image, Image frameImage)
```

Comment:
Is it convenient to have a function called DoLoop? :laughing:  Maybe name it more specifically.